source/docs/hardware/hardware-basics/status-lights-ref.rst

```diff
@@ -5,6 +5,8 @@ Status Light Quick Reference
 
 Many of the components of the FRC\ |reg| Control System have indicator lights that can be used to quickly diagnose problems with your robot. This guide shows each of the hardware components and describes the meaning of the indicators. Photos and information from Innovation FIRST and Cross the Road Electronics.
 
+`Here <https://wpilib.org/s/Status-Light-Reference.pdf>`_ is a compact and printable version of this information.
```

Comment:
It's better if you link a key phase rather than just "here".  What about "A (compact and printable status light reference) is available." (and link the words in parentheses)